Description: additional patch to fix Chromium code for compatibility with Python 3
Origin: https://raw.githubusercontent.com/Homebrew/formula-patches/7ae178a617d1e0eceb742557e63721af949bd28a/qt5/qt5-webengine-chromium-python3.patch
Last-Update: 2024-01-05

--- a/src/3rdparty/chromium/sandbox/policy/mac/package_sb_file.py
+++ b/src/3rdparty/chromium/sandbox/policy/mac/package_sb_file.py
@@ -37,14 +37,14 @@ def pack_file(argv):
   output_h_file = output_directory + '/' + input_basename + '.h'
   output_cc_file = output_directory + '/' + input_basename + '.cc'
   try:
-    with open(input_filename, 'rb') as infile:
-      with open(output_h_file, 'wb') as outfile:
+    with open(input_filename, 'r') as infile:
+      with open(output_h_file, 'w') as outfile:
         outfile.write(header)
         outfile.write(h_include)
         outfile.write(namespace)
         outfile.write(h_definition % module_name)
         outfile.write(namespace_end)
-      with open(output_cc_file, 'wb') as outfile:
+      with open(output_cc_file, 'w') as outfile:
         outfile.write(header)
         outfile.write(cc_include % module_name)
         outfile.write(namespace)
